contacts/html-helper.js

```diff
@@ -0,0 +1,239 @@
+  image.className = CALSSES.AVATAR;
+  parent.appendChild(image);
+};
+
+const createCheckboxPersonHTML = (parent, person) => {
+  const checkbox = document.createElement(TAGS.INPUT);
+  checkbox.type = TYPES.CHECKBOX;
+  checkbox.custom = CUSTOMS.CHECKBOX;
+  checkbox.id = person.checkBoxId;
+  checkbox.disabled = false;
+  checkbox.checked = person.isChecked;
+  parent.appendChild(checkbox);
+};
+
+const createFullNamePersonHTML = (parent, person) => {
+  const fullName = document.createElement(TAGS.SPAN);
+  fullName.id = person.fullNameId;
+  fullName.className = CALSSES.FULL_NAME;
+  const firstName = document.createElement(TAGS.SPAN);
+  firstName.innerHTML = person.firstName;
+  firstName.className = CALSSES.MARGIN_RIGHT;
+  fullName.appendChild(firstName);
+  const lastName = document.createElement(TAGS.SPAN);
+  lastName.innerHTML = person.lastName;
+  fullName.appendChild(lastName);
+  parent.appendChild(fullName);
+};
+
+const createTrashPersonHTML = (parent, person) => {
+  const trash = document.createElement(TAGS.IMAGE);
+  trash.id = person.trashId;
+  trash.src = IMAGES.TRASH;
+  trash.alt = ALT.TRASH;
+  trash.custom = CUSTOMS.TRASH;
+  trash.className = CALSSES.TRASH;
+  parent.appendChild(trash);
+};
+
+const addToHTMLNewPerson = (person) => {
+  const newLi = createListItemPersonHTML(getElementById(DOCUMENT_ELEMENTS.CONTACTS.CONTACTS_LIST), person);
+  createPlusButtonPersonHTML(newLi, person);
+  createCountLikesPersonHTML(newLi, person);
+  createMinusButtonPersonHTML(newLi, person);
+  createAvatarImagePersonHTML(newLi, person);
+  createCheckboxPersonHTML(newLi, person);
+  createFullNamePersonHTML(newLi, person);
+  createTrashPersonHTML(newLi, person);
+  getElementById(DOCUMENT_ELEMENTS.CONTACTS.CONTACTS_LIST).appendChild(newLi);
+};
+
+const clearContactsListHTML = () => {
+  getElementById(DOCUMENT_ELEMENTS.CONTACTS.CONTACTS_LIST).innerHTML = EMPTY;
+};
+
+
+export const populateContactsListHTML = () => {
+  ContactsService.internalArrayContacts.forEach(element => addToHTMLNewPerson(element));
+};
+
+
+const reDrawContactsList = (person = ZERO, target = ZERO) => {
+  switch (target) {
+    case CUSTOMS.MINUS_BUTTON:
+      document.getElementById(person.likesCountId).innerHTML = person.likes;
+      document.getElementById(person.minusButtonId).disabled = isCountEqualToMin(person);
+      document.getElementById(person.plusButtonId).disabled = isCountEqualToMax(person);
+      break;
+    case CUSTOMS.PLUS_BUTTON:
+      document.getElementById(person.likesCountId).innerHTML = person.likes;
+      document.getElementById(person.minusButtonId).disabled = isCountEqualToMin(person);
+      document.getElementById(person.plusButtonId).disabled = isCountEqualToMax(person);
+      break;
+    case CUSTOMS.TRASH:
+      document.getElementById(person.listItemId).remove();
+      break;
+    default:
+      break;
+  }
+};
+
+
+export const reRenderContactsListHTML = (person = ZERO, tarsget = ZERO) => {
```

Comment:
don't know if it is a typo or not, if it is a typo maybe you meant 'target', if not, please add a more suggestive name

Reply:
moreover, the default values are not that fine, person is an object most likely, and initializing it with 0 is not a good idea, only if it was a number